app/controllers/my-projects/project/hearing/add.js

```diff
@@ -0,0 +1,80 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import moment from 'moment';
+
+// create a hearing record when a user clicks on the submit hearing button
+function createHearing(thisHearing, hearingLocation, hearingDate) {
+  const newHearing = thisHearing.store.createRecord('hearing', {
+    project: thisHearing.model,
+    location: hearingLocation,
+    date: hearingDate,
+  });
+  newHearing.save();
+}
+
+export default class MyProjectsProjectHearingAddController extends Controller {
+  @service
+  store;
+
+  // for checking if user clicks submit button without inputting necessary attributes
+  dateMissing;
+
+  timeMissing;
+
+  locationMissing;
```

Comment:
I'd set defaults for these, or make them computed from the model directly

Reply:
Okay sounds good